### DISCUSSION TITLE
Lesson 11 _safeMint error

### DISCUSSION THREAD

**Discussion by Grahovats** [Cyfrin/foundry-full-course-cu#969] [Q&A]
![image (1)](https://github.com/Cyfrin/foundry-full-course-f23/assets/123510382/da40c4ed-997e-4360-9fd1-72d83bc7e115)

**Comment by w3rc** [comment]
You have to decalre the contract as `contract BasicNFT is ERC721`

**Reply by Grahovats** [reply]
Thanks, stupid mistake.

**Comment by Amio13** [comment]
![Screenshot_2023-10-19_155920](https://github.com/Cyfrin/foundry-full-course-f23/assets/117745343/bccdb66d-e4ab-4cc3-a9e5-1d2faa4924eb)
What may be the cause ...since I am new to SC development
